packages/shared/components/popups/StorageDepositBreakdownPopup.svelte

```diff
@@ -1,10 +1,33 @@
 <script lang="typescript">
-    import { HR, BalanceSummarySection, Text, FontWeight, Button } from 'shared/components'
+    import { closePopup, openPopup } from '@auxiliary/popup'
     import { selectedAccount } from '@core/account'
     import { localize } from '@core/i18n'
-    import { closePopup, openPopup } from '@auxiliary/popup'
     import { checkActiveProfileAuth } from '@core/profile'
+    import { Output, selectedAccountAssets } from '@core/wallet'
     import { consolidateOutputs } from '@core/wallet/actions/consolidateOutputs'
+    import { getStorageDepositFromOutput } from '@core/wallet/utils/generateActivity/helper'
+    import { BalanceSummarySection, Button, FontWeight, HR, Text } from 'shared/components'
+
+    $: ({ baseCoin } = $selectedAccountAssets)
+
+    let accountBalance
+    $: $selectedAccount, void getAccountBalance()
+    async function getAccountBalance(): Promise<void> {
+        accountBalance = await $selectedAccount.getBalance()
+    }
+
+    let potentiallyLockedOutputsStorageDeposit
+    $: accountBalance, void calculatePendingTransactionStorageDeposit()
+    async function calculatePendingTransactionStorageDeposit(): Promise<void> {
+        potentiallyLockedOutputsStorageDeposit = 0
+        for (const [outputId, unlocked] of Object.entries(accountBalance?.potentiallyLockedOutputs ?? {})) {
+            if (!unlocked) {
+                const output: Output = (await $selectedAccount.getOutput(outputId)).output
```

Comment:
```suggestion
                const output = (await $selectedAccount.getOutput(outputId)).output as Output
```
I'm getting warnings in vscode because `(await $selectedAccount.getOutput(outputId)).output` returns `OutputTypes` and not `Output`. The two types barely differ. `OutputTypes` possible includes `ITreasuryOutput` whereas the `Output` type doesn't. I suggest casting the type onto the `output` variable to make the typescript error go away.